### PR TITLE
docs: add Sajee to GDE resources

### PR DESCRIPTION
**Pull request by sajeetharan**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
This adds [Sajeetharan Sinnathurai](https://twitter.com/kokkisajee) to the list of Angular GDEs on the Angular Docs page -> cc @StephenFluin

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by sajeetharan** [comment]
I signed it!

> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here (e.g. `I signed it!`) and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google.com/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -663,5 +663,14 @@
     "bio":
       "Kevin is a Angular Taiwan, Angular Girls Taiwan community organzier. He loves sharing knowledge with other developers through blogging, speaking, workshops.",
     "group": "GDE"
+  },
+
+  "sajee": {
+    "name": "Sajeetharan Sinnathurai",
+    "picture": "sajee.jpg",
+    "twitter": "https://twitter.com/kokkisajee",
+    "website": "http://sajeetharan.herokuapp.com/",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: `HTTPS` is generally preferrable :grin:

**Review by StephenFluin — APPROVED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (sajeetharan) [reply aio/content/marketing/contributors.json]: Fixed 👍

**Comment by gkalpak** [comment]
@sajeetharan, can you please squash the commits together and change the commit message to follow [our guidelines][1]? E.g.:

> docs: add Sajee to contributors 

[1]: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Comment by sajeetharan** [comment]
> @sajeetharan, can you please squash the commits together and change the commit message to follow [our guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)? E.g.:
> 
> > docs: add Sajee to contributors

Should be good now

**Comment by gkalpak** [comment]
A space is missing after `docs:` (`docs:add` --> `docs: add`), which makes CI unhappy 😇

**Comment by sajeetharan** [comment]
> A space is missing after `docs:` (`docs:add` --> `docs: add`), which makes CI unhappy 😇

I see you've done the changes. do you need any changes further?

**Comment by gkalpak** [comment]
I fixed the commits (and also took the liberty to fix some whitespace issues in the previous entry :grin:).
It should be good to go now 😅

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**